fs/expose/dokan/__init__.py

```diff
@@ -76,7 +85,9 @@
 from fs.wrapfs import WrapFS
 
 try:
-    import libdokan
+    if six.PY2: import libdokan
+    elif six.PY3: from . import libdokan
```

Comment:
I think you need to change  `from __future__ import with_statement` to `from __future__ import with_statement, absolute_import`
Then change
```
    if six.PY2: import libdokan
    elif six.PY3: from . import libdokan
    else: raise
```
to just
```
    from . import libdokan
```

... I *think* :laughing: